README.md

```diff
@@ -1,157 +1,144 @@
+  ```javascript
+  {
+    id: string;
+  }
+  ```
 
-Desktop version:
-![Desktop version](./desktop-layout.png)
-
-Mobile version:
-![Mobile version](./mobile-layout.png)
+  response
 
-### Display the help text
+  ```javascript
+  {
+    queueNumber: number;
+  }
+  ```
 
-To display the help text, we provide two endpoints on localhost:4000 ( for more details you can check server.js file ):
-- 'api/question' which responses is 
-```
-{ text: 'Question: Can you help me?' }
-```
-- 'api/answer' which response is 
-```
-{ text: 'Answer: Yes sure.' }
-```
+### Layout
 
-When the user clicks the help link, we need to display an alert with the following text:
+The template requires some css rules in order to be displayed as proposed in the screenshots provided. Feel free to use any property you feel like to achieve so.
 
-*Question: Can you help me?.</br>Answer: Yes, sure*
+#### Desktop version
 
-You will have to consider the following:
-- Each endpoint takes 10 seconds to respond, so it will be desirable to avoid the user to wait for this 10 seconds every time the help link is clicked. 
-- The requests can return a 500 http status, in this case you will have to display the message error provided in the response
+![Desktop version](./desktop-layout.png)
 
-## Development
+#### Mobile version
 
-### Prerequisites
+![Mobile version](./mobile-layout.png)
 
-You will need to install on your machine:
-- [NodeJs](https://nodejs.org/en/) in order to run the application.
-- [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) in order to clone the repository containing this challenge and submit the pull request with your solution.
+## Your solution
 
-### Configuration to run the environment
+### Must have
 
-By executing the following lines of code in your terminal you will be able to:
-- Clone the repository containing this challenge.
-- Install the necessary dependencies to run the application.
-- Run the application.
+- README file containing instructions for the project setup along with the chosen stack
 
-```shell
-git clone https://github.com/Creditas/frontend-challenge-vlc.git
-cd frontend/
-npm install
-npm start
-```
+### Extra points
 
-If everything is correct, you should be able to access the following URL: [http://localhost:4000/](http://localhost:4000/).
+- Extra explanation in your README file such as architectural overview, testing strategy, things to improve (next steps) etc
 
-### Dependencies
+### Stack
 
-This project uses some libraries to assist us with tests and running the application. Those are: 
-- [Webpack](https://webpack.js.org/)
-- [Babel](https://babeljs.io/)
-- [Jest](https://jestjs.io/)
-- [Eslint)[https://eslint.org/]
+- You are free to choose the stack and language that you feel comfortable with!
+- In case of doubt which one to choose, we prefer React over Vue or Angular since that's the stack we are currently using at Creditas
```

Comment:
I am not sure about that sentence. Indirectly we are forcing candidates to do the challenge in React because "that is what we prefer". Moreover, in the first sentence we are talking about choose the stack that you want and in that second one we are indicated 3 frameworks 🤔

Reply:
It might seem like forcing, but I see it more like an orientation. Still, I prefer to give a set of preferred frameworks.

In that scenario, a jQuery solution is totally valid and we might not be able to evaluate what we want to evaluate compared to a solution based on those frameworks.

I would choose based on what puts the candidate in a better position for success.

Reply:
Clearly we have different point of view of what is best for a candidate and it's great to have different opinions. This would be a great topic to discuss with a beer, if it weren't for the damn covid 😞

## tldr
For me is ok to have that sentence, but definitely one of the two sentences should be deleted, we can't say choose the stack you want and in the following sentence say we prefer React. 

## Why I don't think that is the best for a candidate?
The first point is that frontend isn't only React, Angular or Vue. Exists a lot of frameworks ember, meteor, mithril... Moreover exists a lot of great engineers that work with vanilla JS or even jQuery and most of them don't know anything about React, Angular or Vue. Really we want to discard that people? 

For me a great engineers isn't that who knows a framework. In fact, for me that is one of the things that I found least valuable when looking for a candidate. In a candidate I look to how she thinks, why she applies that solution to resolve that concrete problem? How the code is organize? Is she applying good pattern to resolve the problem? That are better things to look in a candidate better that if she can user one or another FW. That is the reason why I don't like to put anything about a framework in a challenge.

But as I said before, that is just my opinion and I don't have any problem if we decided to leave that mention to the frameworks. We can always receive feedback from candidates and iterate to improve our challenge 💪

Reply:
I totally understand your point of view.

Based on the discussion, I changed my mind and I would go first with the most freedom for the candidates instead of suggesting what to use (I assumed an issue without any fact backing it), and based on feedback we iterate 😄

Reply:
We are going to keep it open, and not hint about any specific framework.
We are not going to provide a setup of the challenge since the we want to give freedom to candidates to choose their stack.

Thanks for all your suggestions!